ci/lint.sh

```diff
@@ -90,13 +90,26 @@ if [ "$LINT" ]; then
     #
     # Check the following functions:
     # any(), all(), sum(), max(), min(), list(), dict(), set(), frozenset(), tuple(), str.join()
-    grep -R --include="*.py*" -E "[^_](any|all|sum|max|min|list|dict|set|frozenset|tuple|join)\(\[.* for .* in .*\]\)"
+    grep -R --include="*.py*" -E "[^_](any|all|sum|max|min|list|dict|set|frozenset|tuple|join)\(\[.* for .* in .*\]\)" *
```

Comment:
I've added a `*` in the end because without it this regexp will just wait for stdin (unless linux version works differently. I've tried it on mac).

Reply:
+1. On Linux, it will grep all directories without the `*`, but for consistency adding `*` or `.` is good.